### PR TITLE
PSR Coding standards

### PR DESCRIPTION
**Pull request by d1am0nd**
PSR Coding standards

I went through the changes manually so I might've missed some. But should be a good start!

### CODE REVIEW

File: src/Player.php [hunk 1]
```diff
@@ -44,15 +43,15 @@ abstract class Player
 
     public function __construct(string $playerName, string $endpoint)
     {
-
         $this->playerName = $playerName;
 
         $this->client = new Client([
-            'http_errors'   =>  false
+            'http_errors' => false,
```
Comment (d1am0nd) [inline src/Player.php]: Trailing commas are great for minimizing git diffs in PRs. I don't think it's a part of the PSR standard, but definitely the most common coding style

File: src/OSPlayer.php [hunk 2]
```diff
@@ -2,12 +2,10 @@
 
 namespace RunescapeTracker\RunescapePlayerApi;
```
Comment (d1am0nd) [inline src/OSPlayer.php]: PSR: 1 line after namespace declaration

File: src/Player.php [hunk 3]
```diff
@@ -18,8 +17,8 @@ abstract class Player
      * @var array
      */
     protected $player = [
-        'skills'    =>  [],
-        'activities'    =>  []
+        'skills' => [],
```
Comment (d1am0nd) [inline src/Player.php]: Not a PSR standard, but the most common way to deal with arrays is to use exactly 1 space. 2nd most common is to align them - which can be extremely frustrating in git diffs when you add / remove the longest line and you have to adjust the whole array.

File: src/Player.php [hunk 4]
```diff
@@ -44,15 +43,15 @@ abstract class Player
...
-
         $this->playerName = $playerName;
 
         $this->client = new Client([
-            'http_errors'   =>  false
+            'http_errors' => false,
         ]);
 
-        if(array_key_exists($endpoint, $this->endpoints))
+        if (array_key_exists($endpoint, $this->endpoints)) {
```
Comment (d1am0nd) [inline src/Player.php]: The opening bracket on the same line as `if` + there should be a space after `if`

File: src/Player.php [hunk 5]
```diff
@@ -113,10 +108,11 @@ public function player(): array
      * @param $name
      * @return array|null
      */
-    public function getSkill($name)
+    public function getSkill($name): ?array
```
Comment (d1am0nd) [inline src/Player.php]: Typehinting for consistency. The `?` in front indicates the return value can be `null`

File: src/Player.php [hunk 6]
```diff
@@ -127,22 +123,21 @@ public function getSkill($name)
      * @param $name
      * @return array
      */
-    public function getActivity($name): array
+    public function getActivity($name): ?array
```
Comment (d1am0nd) [inline src/Player.php]: The method seems to sometimes return `null`

File: src/Player.php [hunk 7]
```diff
@@ -153,7 +148,8 @@ public function getActivity($name): array
      *
      * @return array
      */
-    public function getSkillList(): array {
+    public function getSkillList(): array
```
Comment (d1am0nd) [inline src/Player.php]: The opening bracket should be on the next line after method declaration

File: src/Player.php (hunk 6 above)
Comment (1e4) [reply src/Player.php]: Nice catch

**Comment by 1e4** [comment]
Changes look great, thanks for the feedback and PR. Do you happen to use PHPStorm? Are there any settings I should set for PSR and standards and such?

**Pull request merged**